src/Subsystems/DriveTrain.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * DriveTrain.cpp
+ *
+ *  Created on: Feb 1, 2016
+ *      Author: Benjamin
+ */
+
+#include <Subsystems/Drivetrain.h>
+#include "../RobotMap.h"
+
+Drivetrain::Drivetrain(RobotDrive drive): Subsystem("Drivetrain") {
+	robot = &drive;
```

Comment:
Make the constructor take a reference to a `RobotDrive` instead of creating a reference in the constructor:

```
Drivetrain::Drivetrain(Robotdrive *drive): 
```